client/trino-client/src/main/java/io/trino/client/StatementClientV1.java

```diff
@@ -108,14 +112,14 @@ class StatementClientV1
     // Data accessor for raw and encoded data
     private final ResultRowsDecoder resultRowsDecoder;
 
-    public StatementClientV1(Call.Factory httpCallFactory, Call.Factory segmentHttpCallFactory, ClientSession session, String query, Optional<Set<String>> clientCapabilities)
+    public StatementClientV1(Call.Factory httpCallFactory, Call.Factory segmentHttpCallFactory, ClientSession session, Optional<String> query, Optional<Set<String>> clientCapabilities)
```

Comment:
I'd revert the Optional<String> change. You can always do `SELECT 1`

Reply:
Reverted via bea011c.